cypress/integration/requests/requests-sort.spec.js

```diff
@@ -52,7 +52,7 @@ describe('ui-requests: Sort requests', () => {
     Requests.deleteRequestPolicyApi(requestPolicyId);
   });
```

Comment:
Please fix linter issue:
'testType' is defined but never used. Allowed unused vars must match /React/u

Reply:
Done